persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/client/ClientDAO.kt

```diff
@@ -18,4 +18,5 @@ interface ClientDAO {
     suspend fun deleteClientsOfUserByQualifiedID(qualifiedID: QualifiedIDEntity)
     suspend fun deleteClient(userId: QualifiedIDEntity, clientId: String)
     suspend fun getClientsOfConversation(id: QualifiedIDEntity): Map<QualifiedIDEntity, List<Client>>
+    suspend fun insertClientsAndRemoveRedundant(qualifiedID: QualifiedIDEntity, clients: List<Client>)
```

Comment:
Missing unit test

Reply:
unitTests are missed for the whole `ClientDAOImpl` ;P